doc/devel/coding_style_guideline.rst

```diff
@@ -7,17 +7,13 @@ DIPY Coding Style Guideline
 The main principles behind DIPY_ development are:
 
 * **Robustness**: the results of a piece of code must be verified
-  systematically, and hence stability and robustness of the code must be
-  ensured, reducing code redundancies.
+  systematically, and hence stability and robustness of the code must be ensured, reducing code redundancies.
```

Comment:
```suggestion
  systematically, and hence stability and robustness of the code must be 
  ensured, reducing code redundancies.
```

Reply:
Was this just an elimination of the new line? I wouldn't stretch it out that long.

Reply:
Oh! Thanks for the suggestion